package.json

```diff
@@ -44,7 +44,7 @@
     "less": "2.7.1",
     "less-plugin-clean-css": "1.5.1",
     "ng-file-upload": "12.2.13",
-    "ngeo": "git://github.com/camptocamp/ngeo#2.1.0-rc.7",
+    "ngeo": "git://github.com/camptocamp/ngeo#2.1.1",
```

Comment:
Thanks!
I have upgraded the PR to use the npm package. Ditto for angular-slug.
There are other packages that may be updated, see https://github.com/c2corg/v6_ui/issues/1455 but I don't have time now to make sure the changes are correct.

I will merge when Travis tests are OK.